packages/core/src/RenderingEngine/VideoViewport.ts

```diff
@@ -82,6 +88,26 @@ class VideoViewport extends Viewport implements IVideoViewport {
     this.videoElement.remove();
   }
 
+  public setVideoImageId(
+    imageIds: string | string[],
+    currentImageIdIndex = 0
+  ): Promise<unknown> {
```

Comment:
As this is a video specific method, I don't see any harm in not calling the currentImageIdIndex a `timePoint` ? or `frameNumber` ?

Reply:
Used frameNumber - it isn't a time point, although if preferred I could set it up to take both arguments to specify what type of position it is?